src/xenia/kernel/title_id_utils.h

```diff
@@ -0,0 +1,78 @@
+/**
+ ******************************************************************************
+ * Xenia : Xbox 360 Emulator Research Project                                 *
+ ******************************************************************************
+ * Copyright 2025 Xenia Canary. All rights reserved.                          *
+ * Released under the BSD license - see LICENSE in the root for more details. *
+ ******************************************************************************
+ */
+
+#ifndef XENIA_KERNEL_UTIL_TITLE_ID_UTILS_H_
+#define XENIA_KERNEL_UTIL_TITLE_ID_UTILS_H_
+
+#include <cstdint>
+#include <utility>
+
+namespace xe {
+namespace kernel {
+
```

Comment:
I think you can put there things like specific IDs that are checked somewhere in the code. Like dashboard id, 0x584D07D1 and so on